src/main/resources/default/assets/tycho/scripts/form.js

```diff
@@ -18,4 +18,51 @@ sirius.ready(function () {
             }
         });
     });
+
+    document.querySelectorAll('.danger-link').forEach(function (_node) {
+        _node.addEventListener('click', function () {
+            if (_node.matches('.guarded-link')) {
+                try {
+                    let modalElement = document.getElementById('danger-link-confirm');
+                    let submitBtn = modalElement.querySelector("button[type='submit']");
+
+                    submitBtn.addEventListener('click', function() {
+                        document.getElementById(_node.getAttribute('data-deleteid')).submit();
+                    });
+
+                    $('#danger-link-confirm').modal({
+                        keyboard: true
+                    }).on('shown', function () {
+                        modalElement.querySelector('.btn-close').focus();
+                    }).attr('tabindex', -1);
```

Comment:
Optional: Vllt stattdessen lieber den neuen weg ohne jQuery nehmen wenn das geht: https://getbootstrap.com/docs/5.0/components/modal/#via-javascript

Reply:
we're still on bs 4, not available

Reply:
😿 Thats why I thought it would be a good idea to directly jump to BS5 to prevent a possible next migration